cmd/jaeger/config-elasticsearch.yaml

```diff
@@ -17,6 +17,19 @@ extensions:
       some_storage:
         elasticsearch:
           index_prefix: "jaeger-main"
+#          indices:
+#            spans:
+#              date_layout: "2006-01-02"
+#              rollover_frequency: "hour"
+#            services:
+#              date_layout: "2006-01-02"
+#              rollover_frequency: "hour"
+#            dependencies:
+#              date_layout: "2006-01-02"
+#              rollover_frequency: "hour"
```

Comment:
Is this actually used? I don't see a cli flag for dependencies index

Reply:
I also found that there is no corresponding cli flag during my change process, so I will add one～

Reply:
ohh, https://github.com/jaegertracing/jaeger/blob/88f856a1a373f954c5f4caec4653932c4f2e74cf/plugin/storage/es/options.go#L369

Because there are no references here, I'll change this to be consistent with the other indexes in this change, okay?